src/lib.rs

```diff
@@ -42,26 +42,27 @@
 //!        let notif = sub.next().await.unwrap();
 //!    }
 //! ```
-
 #![warn(missing_docs)]
 
+mod platform;
 mod utils;
 
-pub use jsonrpsee::{
-    core::client::error::Error as RpcError, rpc_params, types::SubscriptionId,
```

Comment:
can you re-export these again? 

you probably need to add conditional compilation on the PingConfig...

Reply:
`PingConfig` compiles for wasm fine though, would also be good to keep it as well. Ping messages kepp the connection alive

Reply:
WebSocket pings initiated by the client is not possible because the underlying gloo-net doesn't support probably because https://stackoverflow.com/questions/10585355/sending-websocket-ping-pong-frame-from-browser/10586583#10586583.

This will not be used and is not implemeted by jsonrpsee wasm/web client.

However, I think it will reply the server initiated pings. So better to remove it to not give the impression that it is enabled.